cmake/modules/SwiftAndroidSupport.cmake

```diff
@@ -1,16 +1,28 @@
 function(swift_android_libcxx_include_paths var)
-  set(${var}
-        "${SWIFT_ANDROID_NDK_PATH}/sources/cxx-stl/llvm-libc++/include"
-        "${SWIFT_ANDROID_NDK_PATH}/sources/cxx-stl/llvm-libc++abi/include"
-      PARENT_SCOPE)
+  if("${SWIFT_HOST_VARIANT}" STREQUAL "android")
+    set(${var}
+          "${ANDROID_NATIVE_SYSROOT}/usr/include/c++/v1"
+        PARENT_SCOPE)
```

Comment:
I think that we want to ensure that you have the NDK even in the hosted compile mode.

Reply:
See comments above about not using the normal NDK, same for next three comments.